docs/УправлениеИнтеграциями.md

```
@@ -0,0 +1,366 @@
+		
+		Если ОтветHTTP.КодСостояния < 300 Тогда
+			// ...
+		Иначе
+			ТекстСообщения = СтрШаблон("ru = 'Код состояния %1: %2'", ОтветHTTP.КодСостояния,
+				пбп_ИнтеграцииСервер.РасшифровкаКодаСостоянияHTTP(ОтветHTTP.КодСостояния));
+			СтруктураОтвета.ОписаниеОшибки = НСтр(ТекстСообщения);
+		КонецЕсли;
+		
+	Исключение
+		
+		ТекстСообщения = "ru = 'Ошибка при получении компонентов из JIRA'";
```

Markdownlint

Column: 1
Hard tabs

(MD010, no-hard-tabs)